test/client.jl

```diff
@@ -131,14 +131,14 @@ end
         @test status(fetch(t)) == 200
     end
 
-    @testset "Client Redirect Following - $read_method" for read_method in ["GET", "HEAD"]
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/redirect/1")) ==200
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/redirect/1", redirect=false)) == 302
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/redirect/6")) == 302 #over max number of redirects
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/relative-redirect/1")) == 200
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/absolute-redirect/1")) == 200
-        @test status(HTTP.request(read_method, "$sch://httpbin.org/redirect-to?url=http%3A%2F%2Fgoogle.com")) == 200
-    end
+    # @testset "Client Redirect Following - $read_method" for read_method in ["GET", "HEAD"]
```

Comment:
Is there a reason these tests are commented out and not set to `test_broken / test_skip`?

Reply:
Ah, I wasn't aware of `@test_skip`; just pushed that change.

Reply:
You can also do the following to skip the entire set alternatively.

```
@test_skip @testset "Client Redirect Following...."
```